web/src/app/documentation/components/IntegrationKeyAPI.js

```diff
@@ -5,24 +5,16 @@ import Typography from '@material-ui/core/Typography'
 import markdownText from '../IntegrationKeys.md'
 import Markdown from '../../util/Markdown'
 
-function replaceAll(target, search, replacement) {
-  return target.split(search).join(replacement)
-}
-
-const replaceString = 'https://<example.goalert.me>'
-
 export default function IntegrationKeyAPI() {
-  const protocol = window.location.protocol || 'https:'
-  const host = window.location.host
+  const finalText = markdownText.replaceAll(
+    'https://<example.goalert.me>',
+    window.location.origin,
+  )
```

Comment:
Not necessarily introduced by this PR, but this should be using the PublicURL from config, must have been missed when the HTTP prefix work was added

```suggestion
  const [pubURL] = useConfigValue('General.PublicURL')
  const finalText = pubURL
    ? markdownText.replaceAll('https://<example.goalert.me>', pubURL)
    : markdownText
```

Reply:
That was my intention, although as an alternative we could fall back to `origin + pathPrefix`, I'll let you decide which.

IIRC we don't have a way to get the generated/fallback PublicURL from the backend so that would be the next best thing since the prefix is embedded on the page.